release-notes/3.0/preview/api-diff/preview4/3.0-preview4_System.Net.Security.md

```diff
@@ -0,0 +1,349 @@
+# System.Net.Security
+
+``` diff
+ namespace System.Net.Security {
+     public class SslStream : AuthenticatedStream {
++        public virtual TlsCipherSuite NegotiatedCipherSuite { get; }
+     }
++    public enum TlsCipherSuite : ushort {
```

Comment:
cc @krwq, @bartonjs to double check.

Reply:
Looks correct but I think it would be better to copy hex values from here:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Security/src/System/Net/Security/TlsCipherSuite.cs#L18

they look tiny bit better than those generated

Reply:
This diff isn't manually generated, so I would avoid making manual fixups, especially if its primarily cosmetic, imo. The comments and hex value readability don't really provide much value here to justify manual changes like this given the intent/purpose of the diff as part of release notes. The main concern is correctness and potentially missing/incorrect APIs.

Reply:
LGTM in that case